includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -437,6 +438,18 @@ class="location-input wc-enhanced-select dropdown"
 				<option value="physical" <?php selected( $product_type, 'physical' ); ?>><?php esc_html_e( 'I plan to sell physical products', 'woocommerce' ); ?></option>
 				<option value="virtual" <?php selected( $product_type, 'virtual' ); ?>><?php esc_html_e( 'I plan to sell digital products', 'woocommerce' ); ?></option>
 			</select>
+
+			<input
+				type="checkbox"
+				id="woocommerce_sell_in_person"
+				name="sell_in_person"
+				value="yes"
+				<?php checked( $sell_in_person, true ); ?>
+			/>
+			<label class="location-prompt" for="sell_in_person">
+				<?php esc_html_e( 'I will also be selling physical products in person.', 'woocommerce' ); ?>
```

Comment:
Can we change this to read "I will also be selling products or services in person."?

"Physical products" is a little too specific and excludes merchants selling services, tickets, etc. that they might be using Square POS for but wouldn't consider a physical product.